src/components/views/messages/MImageBody.js

```diff
@@ -122,6 +128,10 @@ export default class MImageBody extends React.Component {
         );
     }
 
+    _toggleCollapsed() {
+        this.setState({ collapsed: !this.state.collapsed });
```

Comment:
We'll need to call `this.props.onHeightChanged()` after this state change has rendered (comparing `collapsed` between `prevState` and current state in `componentDidUpdate`), to let the timeline code know this tile changed height. We do this to prevent potential scroll jumps afterwards.